README.md

```diff
@@ -7,90 +7,13 @@ An accessibility visualization toolkit
 [Try tota11y in your browser](http://khan.github.io/tota11y/#Try-it), or
 [read why we built tota11y](http://engineering.khanacademy.org/posts/tota11y.htm).
 
-## Installation
+## Deprecation Notice
 
-`npm install @khanacademy/tota11y`
+tota11y was created at a time when tooling to assist in developing accessible solutions was sparse. Since then, not only have some great browser extensions and automated tooling come into existence, but browsers have added specific development tooling to support accessibility. As such, we are no longer actively developing or maintaininet tota11y.
```

Comment:
I think it would be good to provide a link to Axe here, as that's the defacto standard now. You can link to: https://www.deque.com/axe/ and https://github.com/dequelabs/axe-core